plugins/module_utils/scaleway.py

```diff
@@ -120,7 +120,6 @@ def _url_builder(self, path, params):
 
     def send(self, method, path, data=None, headers=None, params=None):
         url = self._url_builder(path=path, params=params)
-        self.warn(url)
```

Comment:
This change should probably have a changelog fragment. It might be better to actually move it to a separate PR. It looks like a bug to me.